pkg/registry/scheduling/priorityclass/strategy.go

```diff
@@ -43,19 +42,10 @@ func (priorityClassStrategy) NamespaceScoped() bool {
 }
 
 // PrepareForCreate clears the status of a PriorityClass before creation.
-func (priorityClassStrategy) PrepareForCreate(ctx context.Context, obj runtime.Object) {
-	pc := obj.(*scheduling.PriorityClass)
-	pc.Generation = 1
```

Comment:
I was wondering why we'd like to remove these codes, it doesn't look like a problem.

Reply:
Thanks. Restore the initialization of generation value